docsrc/imap/rfc-support.rst

```diff
@@ -390,6 +390,10 @@ The following is an inventory of RFCs supported by Cyrus IMAP.
 
     IMAP URL Scheme, updated by :rfc:`5593`.
 
+:rfc:`5051`
+
+    i;unicode-casemap - Simple Unicode Collation Algorithm
+
```

Comment:
If RFC 5051 can be supported in the context of Sieve and in the context of IMAP, I suggest adding a note, that narrows the scope to Sieve.

Reply:
Fixed